cellar_rebalancer/src/commands/single_signer.rs

```diff
@@ -1,103 +1,55 @@
-            let client = Arc::new(client);
-            let mongo = config.mongo.clone();
-            let name = &config.keys.rebalancer_key;
-            let cosmos_key = config.load_deep_space_key(name.clone());
-
-            let poller = Poller::new(&cellar, client, &mongo, &cosmos_key, config.clone())
-                .await
-                .unwrap_or_else(|e| {
-                    status_err!("couldn't initialize poller: {}", e);
-                    std::process::exit(1);
-                });
-
-            pollers.push(poller);
-        }
-
-        pollers
-    }
-}
+pub struct SingleSignerCmd;
 
 impl Runnable for SingleSignerCmd {
     /// Start the application.
     fn run(&self) {
-        info!("Starting application");
-
         let config = APP.config();
+        info!("Starting application");
         abscissa_tokio::run(&APP, async {
-            let collector =
-                ServiceBuilder::new()
-                    .buffer(20)
-                    .service(Collector::new(&config).unwrap_or_else(|e| {
-                        status_err!("couldn't initialize collector service: {}", e);
-                        std::process::exit(1);
-                    }));
-
-            let pollers = self.build_pollers(config.as_ref().clone()).await;
-
-            let mut tasks = vec![];
-            for poller in pollers {
-                let collector = collector.clone();
-                let task = tokio::spawn(async move { poller.run(collector).await });
-                tasks.push(task);
-            }
-            future::join_all(tasks).await;
+            // Reflection required for certain clients to function... such as grpcurl
+            let contents = fs::read("steward_proto/src/prost/descriptor.bin").unwrap();
+            let proto_descriptor_service = tonic_reflection::server::Builder::configure()
+                .register_encoded_file_descriptor_set(contents.as_slice())
+                .build()
+                .unwrap();
+
+            // Configure TLS
+            let cert = tokio::fs::read(&config.tls.server_cert).await.unwrap();
+            let key = tokio::fs::read(&config.tls.server_key).await.unwrap();
+            let server_identity = Identity::from_pem(cert, key);
+            let client_ca_cert = tokio::fs::read(&config.tls.client_ca_cert).await.unwrap();
+            let client_ca_cert = Certificate::from_pem(client_ca_cert);
+            let tls_config = ServerTlsConfig::new()
+                .identity(server_identity)
+                .client_ca_root(client_ca_cert);
+
+            // run it
+            let addr = ([127, 0, 0, 1], 3000).into();
+            info!("listening on {}", addr);
+            tonic::transport::Server::builder()
+                .tls_config(tls_config)
+                .unwrap_or_else(|err| {
+                    panic!("{:?}", err);
+                })
+                .add_service(UniswapV3CellarAllocatorServer::new(
```

Comment:
This isn't actually the single signer mode if it's just connecting back to the cosmos mode handler. If you want a single signer command you'd need to provide an alternative that uses the old direct Ethereum methodology.

Reply:
You're right, I just pasted this to get it to compile because the previous version was also identical to the cosmos_mode command. There is an issue to fix single-signer mode

Reply:
Removed the handler service just in case